policy/modules/system/systemd.te

```diff
@@ -1366,6 +1366,7 @@ allow systemd_ssh_generator_t self:vsock_socket create;
 allow systemd_ssh_generator_t vsock_device_t:chr_file { read_chr_file_perms };
 
 dev_read_sysfs(systemd_ssh_generator_t)
+kernel_request_load_module(systemd_ssh_generator_t)
```

Comment:
Please note kernel is an exempt from alphabetical order and goes first.
https://github.com/fedora-selinux/selinux-policy/blob/rawhide/policy/modules/system/systemd.te#L608